src/DotNetWorker.Grpc/Http/GrpcHttpRequestData.cs

```diff
@@ -153,8 +153,8 @@ private IReadOnlyCollection<IHttpCookie> ToHttpCookies(string cookieString)
             for (int c = 0; c < separateCookies.Length; c++)
             {
                 var splitArray = separateCookies[c].Split("=", StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Use `.Split(",", StringSplitOptions.RemoveEmptyEntries | StringSplitOptions.TrimEntries);` and then you do not need the explicit `Trim` call on each item (and I would imagine that would also save as some string allocation)

Reply:
My bad! As discussed offline, this will cause the cookie value to be also trimmed, which is something we don't want.